src/formattedcode/format_csv.py

```diff
@@ -66,6 +66,9 @@ def write_csv(scanned_files, output_file, *args, **kwargs):
     w.writeheader()
 
     for r in rows:
+        if len(r) == 2:
```

Comment:
What is the meaning of the this line? what is the magic number 2?

Reply:
@pombredanne This will check if the `OrderedDict()` contains only two keys, the keys being `Resource`, and `scan_errors` (the case that occurs when the csv row is empty).
Something like : 
```
OrderedDict([('Resource', u'/srp/srp_vfy.c'), (u'scan_errors', u'')])
```
that will print an empty row like this :
```
/srp/srp_vfy.c,,,,,,
```
whereas a non-empty row would have an OrderedDict like this : 
```
OrderedDict([('Resource', u'/srp/srp_vfy.c'), (u'copyright', u'Copyright 2011-2016 The OpenSSL Project'), (u'start_line', 2), (u'end_line', 2)])
```
and would print a row like this: 
```
/srp/srp_vfy.c,,Copyright 2011-2016 The OpenSSL Project,2,2,,
```

Reply:
Do you want me to add a small comment for better code readability ?

Reply:
Rather than comments, the best would be to use explicit attribute access, named intermediate variable(s) and or function(s) to make this explicit rather than cryptic.